### DISCUSSION TITLE
Help with app manifest - checkver

### DISCUSSION THREAD

**Discussion by hitrikrtek** [ScoopInstaller/Main#1884] [General]
Hi,

It is my first try to make installer for Gajim with my own bucket. I've managed to get the installer to install correctly (it is a Nullsoft installer and it took me a while to get this working actually) - so all is good, probably not the best as I read that scoop doesn't like to run installers but anyway, here's my manifest and I'd like to add checkver for newer version in it and here I have problem, I don't know to do it. Or where to ask for that matter, so if this is not good place, please delete.

This site has note "Latest stable release" "version"
https://gajim.org/download/

My manifest looks like this:
```json
{
    "version": "1.3.1",
    "description": "A fully-featured XMPP client",
    "homepage": "https://gajim.org/",
    "architecture": {
        "64bit": {
            "url": "https://gajim.org/downloads/1.3/Gajim-1.3.1-64bit-1.exe#/gajim-setup.exe",
            "hash": "md5:000FAD1E8334AE94A3B605DFC8F5A1DE"
        },
        "32bit": {
            "url": "https://gajim.org/downloads/1.3/Gajim-1.3.1-32bit-1.exe#/gajim-setup.exe",
            "hash": "md5:3E501810BF47BC0751794CE61CF5D1B0"
        }
    },
	"installer": {
    	"file": "gajim-setup.exe",
    	"args": [
	      		"/S",
	      		"/D=$dir"
    		]
  	},
  	"uninstaller": {
    		"file": "uninstall.exe",
    		"args": "/S"
  	},
	"bin": [
		[ "bin\\Gajim.exe", "Gajim" ]
	],
	"checkver": {
		"url": "http://gajim.org/download/",
		"regex": "<span class="label label-primary">\\s*"([^\"]+)\"
	}
}
```
The regex part is not done correctly, I have no skill with regex so I have no clue... without the checkver part it all works nice.

Thanks, any help is appreciated.

My bucket:
https://github.com/hitrikrtek/scoop-mole

**Comment by akirayamamoto** [comment]
I suggest you create a pull request and use the portable version. Make sure to add the `checkver` and `autoupdate` fields.

https://scoop-docs.vercel.app/docs/concepts/App-Manifest-Autoupdate.html#using-checkver-ps1-to-query-and-autoupdate
